supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -174,7 +174,7 @@ Capitalize it only when it starts a sentence.
 
 [[identity-provider-federation]]
 ==== image:images/yes.png[yes] identity provider federation (noun)
-*Description*: In Red{nbsp}Hat Single Sign-On, you can delegate authentication to one or more IDPs, and this process is referred to as _identity provider federation_. Social login through Facebook or Google+ is an example of identity provider federation. You can also hook Red{nbsp}Hat Single Sign-On to delegate authentication to any other OpenID Connect or SAML 2.0 IDP.
+*Description*: In Red{nbsp}Hat Single Sign-On, you can delegate authentication to one or more IdPs, and this process is referred to as _identity provider federation_. Social login through Facebook or Google+ is an example of identity provider federation. You can also hook Red{nbsp}Hat Single Sign-On to delegate authentication to any other OpenID Connect or SAML 2.0 IdP.
```

Comment:
To stop the Vale issue, update the https://github.com/aireilly/supplementary-style-guide/blob/main/.vale/styles/config/vocabularies/SSG/accept.txt#L70 and remove the IDP entry. 

Aidan raised a higher-level Vale update with https://github.com/redhat-documentation/vale-at-red-hat/issues/850 but this issue would require a bit of work as all terms need to be analyzed for input into the Vale project.